src/components/MarkdownTextArea/index.jsx

```diff
@@ -0,0 +1,138 @@
+import classNames from 'classnames';
+import { string, func, number } from 'prop-types';
+import React, { Component } from 'react';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import Markdown from '../Markdown';
+
+@withStyles(theme => {
+  const borderColor =
+    theme.palette.type === 'light'
+      ? fade(theme.palette.common.black, 0.23)
+      : fade(theme.palette.common.white, 0.23);
+
+  return {
+    tab: {
+      flexGrow: 1,
+      color: theme.palette.text.primary,
+    },
+    tabContent: {
+      marginTop: theme.spacing.unit,
+      padding: theme.spacing.unit,
+    },
+    markdownContainer: {
+      borderColor,
+      borderWidth: 1,
+      borderRadius: theme.shape.borderRadius,
+      borderStyle: 'solid',
+    },
+    markdown: {
+      '& > :first-child': {
```

Comment:
I was able to remove the margin-top with that :)

Reply:
Ah, I see now. Thanks for the help in closing. And sorry I didn't wrap this up sooner.

Reply:
No worries, I really appreciate the good work you've been doing :)